### DISCUSSION TITLE
ffmpeg --HEAD always can't compile successfully

### DISCUSSION THREAD

**Discussion by eko5624** [Homebrew/discussions#3092] [Everyday usage]
brew install ffmpeg --HEAD
https://github.com/eko5624/FFmpeg-macos/actions/runs/1997093808/workflow
==> Installing ffmpeg --HEAD
==> ./configure --prefix=/usr/local/Cellar/ffmpeg/HEAD-c834083 --enable-shared -
==> make install
Last 15 lines from /Users/runner/Library/Logs/Homebrew/ffmpeg/02.make:
POD	doc/libavcodec.pod
POD	doc/libavformat.pod
POD	doc/libavdevice.pod
POD	doc/libavfilter.pod
GEN	libavfilter/libavfilter.ver
LD	libswscale/libswscale..dylib
LD	libpostproc/libpostproc..dylib
ld: malformed 32-bit x.y.z version number: -exported_symbols_list
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [libswscale/libswscale..dylib] Error 1
make: *** Waiting for unfinished jobs....
ld: malformed 32-bit x.y.z version number: -exported_symbols_list
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [libpostproc/libpostproc..dylib] Error 1
rm libavfilter/metal/vf_yadif_videotoolbox.metal.air libavfilter/metal/vf_yadif_videotoolbox.metallib libavfilter/metal/vf_yadif_videotoolbox.metallib.c

Didn't work. Any help would be great, thanks.

**Comment by SMillerDev** [comment]
HEAD builds use the latest upstream commit. Those don't always work, which is to be expected.

**Comment by eko5624** [comment]
Got it, thanks.
